CONTRIBUTING.md

```diff
@@ -27,9 +37,75 @@ again.
 3. Please include unit tests (and integration tests if applicable) for all new code.
 4. Make sure all existing tests pass.
    * In `jib-core`, run `./gradlew clean goJF build integrationTest`
-   * In `jib-gradle-plugin`, run `./gradlew clean goJF build` *
-   * In `jib-maven-plugin`, run `./mvnw clean fmt:format verify` *
```

Comment:
The ending * was there to refer to the note below.

Reply:
Oh!  I'll call this out explicitly above.